scipy/sparse/linalg/tests/test_propack.py

```diff
@@ -89,6 +89,7 @@ def test_svdp(ctor, dtype, irl, which):
         check_svdp(n, m, ctor, dtype, k, irl, which)
 
 
+@pytest.mark.xslow
```

Comment:
I think I wrote this test, and I don't think there's anything very special about it; it's just one that happened to be distributed with PROPACK. We have other tests that don't take as long.